modin/pandas/test/test_series.py

```diff
@@ -961,15 +959,17 @@ def test_asof(where):
     # With NaN:
     values = [1, 2, np.nan, 4]
     index = [10, 20, 30, 40]
-    modin_series, pandas_series = pd.Series(values, index=index), pandas.Series(
-        values, index=index
+    modin_series, pandas_series = (
+        pd.Series(values, index=index),
+        pandas.Series(values, index=index),
```

Comment:
why do you want to change this in this PR? I agree the changed version looks cleaner when formatted, but it's purely aesthetical and doesn't correlate with the issue being fixed (so looks like an unrelated change being squeezed into).

Reply:
@vnlitvinov This reformatting happened during the use of black. Otherwise, black and flake8 in ci would be red.